screens/auth/OnboardingScreen.js

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Image } from 'react-native';
+import Carousel, { Pagination } from 'react-native-snap-carousel';
+import {
+  Body,
+  ButtonLabel,
+  FilledButtonContainer,
+  Title,
+} from '../../components/BaseComponents';
+import Colors from '../../constants/Colors';
+import {
+  ONBOARDING_CONTENT,
+  ONBOARDING_IMAGES,
+} from '../../constants/Onboarding';
+import { OnboardingContainer } from '../../styled/auth';
+import {
+  JustifyCenterContainer,
+  SpaceBetweenRowContainer,
+} from '../../styled/shared';
+
+export default class OnboardingScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true,
+      pageIndex: 0,
+    };
+  }
+
+  get pagination() {
+    return (
+      // TODO @tommypoa: Move styles to styled folder?
+      <Pagination
+        dotsLength={ONBOARDING_CONTENT.length}
+        activeDotIndex={this.state.pageIndex}
+        containerStyle={{ backgroundColor: Colors.lightest, marginTop: 20 }}
+        dotStyle={{
+          width: 8,
+          height: 8,
+          borderRadius: 5,
+          marginHorizontal: 1,
+          backgroundColor: Colors.primaryGreen,
+        }}
+        inactiveDotStyle={{
+          backgroundColor: Colors.base,
+        }}
+        inactiveDotOpacity={1}
+        inactiveDotScale={1}
+      />
+    );
+  }
+
+  navigateWelcome() {
+    this.props.navigation.navigate('Welcome');
+  }
+
+  navigateLogIn() {
+    this.props.navigation.navigate('LogIn');
+  }
+
+  _renderItem({ item, index }) {
+    return (
+      <JustifyCenterContainer>
+        <Image
+          source={ONBOARDING_IMAGES[index]}
+          style={{ height: 55, width: 190 }}
```

Comment:
should we switch this to %

Reply:
Not a huge concern rn since these are just filler images before the final designs are complete